lib/paperclip.rb

```diff
@@ -129,6 +129,8 @@ module ClassMethods
     #   Paperclip::Attachment#interpolate for more information on variable interpolaton.
     #     :url => "/:class/:attachment/:id/:style_:filename"
     #     :url => "http://some.other.host/stuff/:class/:id_:extension"
+    # Note: When using the +s3+ storage option, the +url+ option expects particular values.
+    # See the Paperclip::Storage::S3#url documentation for specifics.
```

Comment:
These two lines should be intended two spaces if they are meant to extend on the previous point.